sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/FixedWindows.java

```diff
@@ -76,9 +76,27 @@ private FixedWindows(Duration size, Duration offset) {
 
   @Override
   public IntervalWindow assignWindow(Instant timestamp) {
```

Comment:
Should FixedWindowsTest be updated?

Reply:
Yes, we wouldn't want anyone to get surprised at the end of the universe.